tools/deploy_documentation.sh

```diff
@@ -26,4 +26,4 @@ pwd
 # Push to qiskit.org website
 openssl aes-256-cbc -K $encrypted_rclone_key -iv $encrypted_rclone_iv -in tools/rclone.conf.enc -out $RCLONE_CONFIG_PATH -d
 echo "Pushing built docs to website"
-rclone sync --progress ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
+rclone sync --progress --exclude-from ./locale/** ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
```

Comment:
You want to use `--exclude` here, `--exclude-from` is to load a list of exclusion regexes from a file.

```suggestion
rclone sync --progress --exclude ./locale/** ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
```